src/sentry/static/sentry/app/components/events/interfaces/breadcrumbsV2/breadcrumbCategory.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import Tooltip from 'app/components/tooltip';
+import overflowEllipsis from 'app/styles/overflowEllipsis';
+
+type Props = {
+  category?: string;
+};
+
+const BreadcrumbCategory = ({category = 'generic'}: Props) => (
+  <div>
```

Comment:
Is this `div` needed?

Reply:
unfortunately yes, otherwise the tooltip will not be aligned to the center